src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl

```diff
@@ -5857,6 +5859,18 @@ unit:Ci
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
   rdfs:label "Curie"@en .
 
+unit:Ci-PER-KiloGM
```

Comment:
Our naming convention for units is all uppercase, except for prefixes, so it should be CI

Reply:
ah ok